parlai/mturk/core/dev/agents.py

```diff
@@ -46,13 +44,11 @@ def __init__(self, status=None):
             status = self.STATUS_NONE
         self.status = status
         self.messages = []
-        self.last_command = None
```

Comment:
Last command has been removed, as the state on the heroku server has this detail stored locally already